core/src/main/java/edu/wpi/grip/core/operations/networktables/NTPublishOperation.java

```diff
@@ -59,6 +60,10 @@ public NTPublishOperation(Class<S> socketType, Class<T> reportType, Function<S,
                 ntValueMethods.add(method);
             }
         }
+
+        // Sort the accessors by their weight. As long as they all have unique weights, this lets us avoid the issue
+        // of different JVMs returning methods in different orders.
+        ntValueMethods.sort(Comparator.comparing(method -> method.getAnnotation(NTValue.class).weight()));
```

Comment:
Throw an exception if there is duplicate weights and write a test that ensures that this happens.